R/durbinWatsonTest_tidiers.R

```diff
@@ -3,7 +3,7 @@
 #' Tidies Durbin-Watson Test objects, from the \code{car} package
 #' into a one-row data frame.
 #'
-#' @param x An object of class \code{"durbinWatsonTest"}
+#' @param x An object of class \code{durbinWatsonTest}
 #' @param ... extra arguments (not used)
 #'
```

Comment:
Just switched over to documenting with rmarkdown, this seems like a good idea to fix now as well: [`rmarkdown`](https://cran.r-project.org/web/packages/roxygen2/vignettes/markdown.html).

Reply:
I'm not sure what needs to be done for this.

Reply:
Instead of using \code{car}, use `car`. It's  a different way to markup the documentation.